_partials/_graphing-ohlcv-data.md

```diff
@@ -0,0 +1,33 @@
+## Graph OHLCV data
+
+When you have extracted the raw OHLCV data, you can use it to graph the result
+in a candlestick chart, using Grafana. To do this, you need to have Grafana set
+up to connect to your TimescaleDB database. For more information about how to do
+this, see the [Grafana setup instructions][grafana-setup].
+
+<Procedure>
+
+### Graphing OHLCV data
+
+1.  Ensure you have Grafana installed, and you are using the TimescaleDB
+    database that contains the Twelve Data stocks dataset set up as a
+    data source. For more information about how to do this, see the [Grafana
+    setup instructions][grafana-setup].
```

Comment:
```suggestion
1.  Install Grafana and the Twelve Data stocks database following the [Grafana
    setup instructions][grafana-setup].
```

Reply:
@Rajakavitha1 It just occurred to me that we might have a partial we can use here. Check through the Grafana partials?

Reply:
Hi @Loquacity I checked the _grafana_vix_prereqs and it reads: Imported the stock trade data from the [Getting Started Tutorial][gsg-data] in this tutorial we are ingesting realtime with a web socket.

Reply:
hrm, OK, no worries. Sorry for the bum steer ☹️